doc/man7/EVP_PKEY-EC.pod

```diff
@@ -70,8 +70,8 @@ I<order> multiplied by the I<cofactor> gives the number of points on the curve.
 
 =item  "decoded-from-explicit" (B<OSSL_PKEY_PARAM_EC_DECODED_FROM_EXPLICIT_PARAMS>) <integer>
 
-Gets a flag indicating whether the key or parameters were decoded from explicit
-curve parameters. Set to 1 if so or 0 if a named curve was used.
+Sets or gets a flag indicating whether the key or parameters were decoded from
+explicit curve parameters. Set to 1 if so or 0 if a named curve was used.
```

Comment:
As I've previously mentioned (in an issue/PR somewhere that I forget) our documentation is very unclear about what parameters are gettable, which are settable, which are importable and which are exportable. This documentation now says the flag can be "set" - but in fact it does not appear in the "settables" list. This change actually makes it importable/exportable.

Reply:
Hmm, right. It is not settable. And IMO should not be. So I am just going to drop this change. It is not my ambition in this PR to fix the documentation of importable parameters.